content/publishing/publish-release-notes.md

```diff
@@ -4,14 +4,42 @@ title: Publish release notes
 weight: 4
 ---
 
-Create a custom release notes file to notify users of the changes as you publish a new version of your app.
+Create custom release notes file(s) to notify users of the changes as you publish a new version of your app.
 
 Release notes can be published to:
 
-* **Email**. The release notes will be included in the publishing email if you have publishing to email configured in App settings > Publish > Email.
-* **Slack**. The release notes will be included in the Slack notification if you have Slack publishing configured in App settings > Publish > Slack.
-* **Google Play**. The release notes will be published to Google Play Console if you have Google Play publishing configured in App settings > Publish > Google Play. The content of the release notes file is published for `en-US`.
+* **Email**. The release notes will be included in the publishing email of a successful build if you have publishing to email configured in App settings > Publish > Email.
+* **Slack**. The release notes will be included in the Slack notification of a successful build if you have Slack publishing configured in App settings > Publish > Slack.
+* **GitHub Releases**. The release notes will be published to a release if you have GitHub releases configured in App settings > Publish > GitHub releases.
+* **Google Play**. The release notes will be published to Google Play Console if you have Google Play publishing configured in App settings > Publish > Google Play.
 
 ## Setting up release notes
 
-Create a `release_notes.txt` file and add it to the root of the repository. When present, Codemagic will fetch the content of that file and publish it with the build.
\ No newline at end of file
+The next 3 options to setup release notes are supported:
+
+1. Create a `release_notes.txt` file and add it to the root of your project. When present, Codemagic will fetch the content of that file and publish it with the build.
+    * To Email, Slack and GitHub releases it will be published as is.
+    * To Google Play it will be published under `en-US` language localization code.
```

Comment:
* For email, Slack and GitHub releases, it will be published as is.
* For Google Play, it will